### PR TITLE
Prepare some issue templates

### PR DESCRIPTION
**Pull request by scholzj**
I prepared some templates for the issues. They include 4 types:
* Bug
* Feature request
* Question
* Docs

Each of them has some simple template and automatically assigned labels. Please let me know if you think something is missing in the templates etc.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Create Custom Resource '....'
+3. Run command '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Desktop (please complete the following information):**
```
Comment (tombentley) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Desktop?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ - Strimzi version: [e.g. master, 0.11.1]
+ - Installation method: [e.g. YAML files, Helm chart, OperatorHub.io]
+ - Kubernetes cluster: [e.g. Kubernetes 1.14, OpenShift 3.9]
+ - Infrastructure: [e.g. Amazon EKS, Minikube]
+
+**Custom Resource YAMLs**
+Attach or copy paste the custom resources you used.
+
+**Logs**
+Attach or copy paste the relevant logs.
```
Comment (tombentley) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
Attach or copy and paste the relevant logs.
```

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+labels: documentation
+assignees: ''
+
+---
+
+**Suggestion / Problem**
+Describe your problem or suggestion
+
+**Documentation Link**
+Provide a link to the documentation where the bud / suggestion should be applied
```
Comment (tombentley) [inline .github/ISSUE_TEMPLATE/documentation.md]:
```suggestion
Provide a link to the documentation where the bug / suggestion should be applied
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (asorokhtey) [reply .github/ISSUE_TEMPLATE/bug_report.md]: yes, maybe "Environment information"?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Desktop (please complete the following information):**
+ - Strimzi version: [e.g. master, 0.11.1]
+ - Installation method: [e.g. YAML files, Helm chart, OperatorHub.io]
+ - Kubernetes cluster: [e.g. Kubernetes 1.14, OpenShift 3.9]
+ - Infrastructure: [e.g. Amazon EKS, Minikube]
```
Comment (asorokhtey) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Maybe we need OS version as well?

Comment (scholzj) [reply .github/ISSUE_TEMPLATE/bug_report.md]: OS version of what? The host OS should not matter. And the container OS should be always ours.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
```
Comment (ppatierno) [inline .github/ISSUE_TEMPLATE/bug_report.md]: What should be an example of "Go to"?

Comment (scholzj) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Go to Grafana Dashboard? Honestly the Bug and Feature request templates are from GitHub. I just deleted the things which were obvious as inapplicable.  So if you have better idea what to put there we should do it.

**Pull request merged**